core/themes/seven/css/style.css

```diff
@@ -1077,27 +1067,59 @@ div.filter-options select {
 /**
  * System.
  */
-div.admin .right,
-div.admin .left {
+.admin .right,
+.admin .left {
   width: 49%;
   margin: 0;
 }
-div.admin-panel,
-div.admin-panel .body {
+.admin-panel,
+.admin-panel .body {
   padding: 0;
   clear: left;
 }
-div.admin-panel {
+.admin-panel {
   margin: 0 0 20px;
   padding: 9px;
   background: #f8f8f8;
   border: 1px solid #ccc;
 }
-div.admin-panel h3 {
+.admin-panel h3 {
+  margin: 4px 0 4px 4px;
+  padding: 6px 0 6px 35px;
   font-size: 0.923em;
   text-transform: uppercase;
-  margin: 0;
-  padding-bottom: 9px;
+  background: url('../images/fa-chevron-circle-right.png') no-repeat left center;
+  background-size: 23px;
+}
+.admin-panel-people h3 {
+  background-image: url('../images/fa-user.png');
+}
+.admin-panel-content h3 {
+  background-image: url('../images/fa-pencil-square-o.png');
+}
+.admin-panel-system h3 {
+  background-image: url('../images/fa-hdd-o.png');
+}
+.admin-panel-administration h3 {
+  background-image: url('../images/fa-cog.png');
+}
+.admin-panel-media h3 {
+  background-image: url('../images/fa-picture-o.png');
+}
+.admin-panel-search h3 {
+  background-image: url('../images/fa-search.png');
+}
+.admin-panel-regional h3 {
+  background-image: url('../images/fa-globe.png');
+}
+.admin-panel-urls h3 {
+  background-image: url('../images/fa-link.png');
+}
+.admin-panel-development h3 {
+  background-image: url('../images/fa-wrench.png');
+}
+.admin-panel-services h3 {
+  background-image: url('../images/fa-rss.png');
```

Comment:
All of these images, I don't think they should be prefixed with "fa" in the event we need to add icons that are provided by Font Awesome (or we change the icons at some point). It might still be good to group them, either with a different prefix or putting them in a subdirectory.

Reply:
> All of these images, I don't think they should be prefixed with "fa"

okay. can remove.

> It might still be good to group them

My first thought was also to use an "icons" subdirectory, but every image in the seven theme is an icon. That would be confusing to people, why are some icons in a subdirectory, and others are not?

Reply:
renamed icons based on growing consensus in that issue.